relay-event-normalization/Cargo.toml

```diff
@@ -22,6 +22,7 @@ relay-base-schema = { path = "../relay-base-schema" }
 relay-common = { path = "../relay-common" }
 relay-event-schema = { path = "../relay-event-schema" }
 relay-log = { path = "../relay-log" }
+relay-metrics = { path = "../relay-metrics" }
```

Comment:
We might want to export `MetricResourceIdentifier` to avoid introducing this dependency.

Reply:
Could you move MRI to `relay-base-schema/metrics` instead?

Reply:
I could, I would expect it to be better to make a separate PR for doing so. WDYT?

Reply:
Sounds good!

Reply:
https://github.com/getsentry/relay/pull/2921